src/ray/gcs/gcs_server/actor_info_handler_impl.cc

```diff
@@ -0,0 +1,54 @@
+#include "actor_info_handler_impl.h"
+#include "ray/util/logging.h"
+
+namespace ray {
+namespace rpc {
+
+void DefaultActorInfoHandler::HandleAsyncGet(const rpc::ActorAsyncGetRequest &request,
+                                             rpc::ActorAsyncGetReply *reply,
+                                             rpc::SendReplyCallback send_reply_callback) {
+  RAY_LOG(DEBUG) << "Begin handle async get, actor_id is:" << request.actor_id();
```

Comment:
This log may be confusing. May be better to use "Begin get actor info, actor_id is: xxxx"?

Reply:
fixed